app/assets/stylesheets/master.css.scss

```diff
@@ -44,7 +44,20 @@
   margin-top: 25px;
 }
 
+.navbar {
+  z-index: +10;
+  margin-bottom: 0px !important;
+}
+
 .navbar-default  .container-fluid {
-  background-image:url(cheap_diagonal_fabric.png); 
+  background-image:url(cheap_diagonal_fabric.png);
+}
+
+html, body {
+  padding: 0; margin: 0;
+}
+
+#earth_div {
+  top: 0px; right: 20px; bottom: 50px; left: 20px; position: relative !important;

```

Comment:
I had removed the important! from a different line and missed this one. Sorry about that!